test/http_client.c

```diff
@@ -28,6 +27,7 @@
 #endif
 #include <sys/stat.h>
 #include <fcntl.h>
+#include <event2/event.h>
```

Comment:
This `#include` seems unnecessary?

Reply:
That's correct. The code that used it got moved to another file.